lib/jelix-www/js/jacldb_admin.js

```diff
@@ -0,0 +1,44 @@
+$("document").ready( function () {
+    if ($("#search-bar").length) {
+        if ($('#search-bar').data('select-on-ac')) {
+            $("#search-bar").autocomplete({
+                source: $("#search-bar").data('link'),
+                select: function (event, ui) {
+                    $("#search-bar").val(ui.item.value.login);
+                    $("#type-hidden").val(ui.item.value.type);
+                    return false;
+                }
+           });
+        } else {
+            $("#search-bar").autocomplete({
+                source: $("#search-bar").data('link'),
+            });
+        }
+    }
+}); 
+
+function hideSelect()
+{
+    if ($("#type-list").children('option:selected').val() === "user") {
+        $("#hideField").show();
+    } else {
+        $("#hideField").hide();
+    }
+}
+
+function showColumn()
+{
+    if (!$("#groupSelector").children("option:selected").length) {
```

Comment:
use variables. don't repeat same jQuery selector calls in your code. `$("#groupSelector").children("option:selected")` is repeated 4 times, it is not efficient. same for `$("#group-head")` etc